tests/zfs-tests/tests/functional/cli_root/zpool_import/zpool_import_missing_003_pos.ksh

```diff
@@ -57,6 +57,10 @@
 #		  before data integrity is compromised
 #
 
+if [[ $TRAVIS = 'true' ]]; then
+	log_unsupported "Test case is disabled in Travis-ci (timeout kill)"
+fi
```

Comment:
We're seeing spurious failures for the same test case in buildbot too.  Why don't we simply disable it across the board for now and open an issue in the tracker so we can track it and get it properly resolved.

Reply:
Agree, https://github.com/zfsonlinux/zfs/pull/6840